src/components/blocks/prefooter-form.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import { t } from '$lib/i18n';
+  import type { ComponentProps } from 'svelte';
+  import PreFooterAsset from '../illustrations/assets/pre-footer.webp';
+  import PreFooterAssetLight from '../illustrations/assets/pre-footer-light.webp';
+  import { theme } from '$lib/stores/theme';
+  import ContactForm from '$components/contact-form.svelte';
+  import type { PrefooterFormStoryblok } from '$types/bloks';
+
+  export let variant: ComponentProps<ContactForm>['variant'] = undefined;
```

Comment:
If this is a block there's no way to pass a variant apart from `export let block`, let it be hardcoded or pass it from storyblok.